public/tier1/utlsymbol.h

```diff
@@ -46,15 +51,32 @@ class CUtlSymbol
 	
 	// operator=
 	CUtlSymbol& operator=( CUtlSymbol const& src ) { m_Id = src.m_Id; return *this; }
-	
+
+	// operator++
+	CUtlSymbol& operator++() { ++m_Id; return *this; }
+	CUtlSymbol operator++(int) { CUtlSymbol oldId = *this; ++m_Id; return oldId; }
+
+	// operator--
+	CUtlSymbol& operator--() { --m_Id; return *this; }
+	CUtlSymbol operator--(int) { CUtlSymbol oldId = *this; --m_Id; return oldId; }
+
 	// operator==
 	bool operator==( CUtlSymbol const& src ) const { return m_Id == src.m_Id; }
 
+	static uint32 Hash( bool bInsensitive, const char *pString, int nLength )

```

Comment:
The name of it isn't quite verbose, how about ``MakeHash``? I'm also not quite sold on the idea of having these static Hash methods and would prefer hashing in place with their dedicated methods, but I'll think a bit on these so leave them as is rn